tools/SourceKit/lib/SwiftLang/SwiftIndexing.cpp

```diff
@@ -135,6 +135,11 @@ class SKIndexDataConsumer : public IndexDataConsumer {
       uidAttrs = getDeclAttributeUIDs(symbol.decl);
       info.Attrs = uidAttrs;
     }
+    if (auto *VD = dyn_cast<ValueDecl>(symbol.decl)) {
+      AccessLevel accessLevel = VD->getEffectiveAccess();
+      UIdent AttrUID = SwiftLangSupport::getUIDForAccessLevel(accessLevel);
+      info.EffectiveAccessLevel = AttrUID;
+    }
```

Comment:
I think the access level info should probably only be added to definitions rather than both the definition and references of the symbol (i.e. move this within the !isRef block above). Is there a particular use case you had in mind for adding it to references as well?

Reply:
No need, it was an oversight. It being on decls is exactly what I need 👍